assets/js/theme/core/amp/AmpProductUtils.js

```diff
@@ -21,9 +21,9 @@ export default class ProductUtils {
     }, options.callbacks);
   }
 
-  init(context) {
+  init(context, productTemplate) {
```

Comment:
NOTE: here we added productTemplate parameter that will be used later on to return part of the product options payload as rendered HTML. This template will be provided during initialising productUtils for AMP pages within specific Theme. For instance, check it on [Peak](https://github.com/bigcommerce/peak/pull/17/files#diff-f79fc70ed1ad087929e106480f7183b5c325b1833c2739fe005b75a3ddbcbafcR20).

Reply:
I have an idea of small optimization. We can set `template` as instance prop on stage of creation like
 ```
this.ProductUtils = new AmpProductUtils(this.el, {
      callbacks: {},
      template: 'products/add-to-cart-form',
    }).init(this.context);
```
   It can help to aviod extra params in `init` and `_bindProductOptionChange` methods

Reply:
done. Thanks for a good point.